examples/objectServerExample/src/main/java/io/realm/examples/objectserver/CounterActivity.java

```diff
@@ -131,6 +132,7 @@ protected void onStop() {
         }
         closeRealm();
         user = null;
+        counter = null;
```

Comment:
showing `removeAllChangeListeners` or `removeChangeListener` is also a good idea instead ? also do we want to call `user.logout` as well?

Reply:
It is on purpose I'm not logging the user out, since there is a manual logout button in the actionbar. It means the next time you open the app you go directly to this Activity.